src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1604,8 +1604,8 @@ private void ApplyAutoEvoResults()
         GD.Print("Applying auto-evo results");
         CurrentGame.GameWorld.GetAutoEvoRun().ApplyExternalEffects();
 
-        CurrentGame.GameWorld.Map.RemoveExtinctSpecies(FreeBuilding);
-
+        var extinct = CurrentGame.GameWorld.Map.RemoveExtinctSpecies(FreeBuilding);
+        extinct.ToList().ForEach(CurrentGame.GameWorld.RemoveSpecies);
```

Comment:
Based on my previous comment this would be at least `extinct.ToList().ForEach`
though in the current codebase .ForEach is not used. So for consistency this could be rewritten as:
```
foreach(var species in extinct)
    CurrentGame.GameWorld.RemoveSpecies(species);
```